client/src/app/components/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,47 @@
+@import "./src/theme.scss";
+
+:host ::ng-deep .mat-sort-header-container {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
Do we absolutely need to use ng-deep here?

Reply:
This whole block has been removed as it is no longer needed since width of 100% is being used for the area graph.

Reply:
If we don't use ng-deep it won't apply the CSS this is because ng-deep forces the CSS to be applied to it's child components. Another reason is some components can't access the elements directly because they are outside of this components so ng-deep takes care of that.

Reply:
yes, but previously, this had impacts on other components in the DOM tree. Can you guarantee that nothing will be affected by this change?

Reply:
I see your point and I completely agree. This is why I cannot guarantee that it will not affect other components so in order to avoid this and to guarantee, I will add a parent class to this CSS so that only this specific CSS block is affected by it.